doc/classes/Light3D.xml

```diff
@@ -88,12 +88,12 @@
 		<member name="light_negative" type="bool" setter="set_negative" getter="is_negative" default="false">
 			If [code]true[/code], the light's effect is reversed, darkening areas and casting bright shadows.
 		</member>
-		<member name="light_projector" type="Texture2D" setter="set_projector" getter="get_projector">
+		<member name="light_projector" type="Texture2D" setter="set_projector" getter="get_projector" keywords="cookie">
```

Comment:
Legitimately curious now. If it wasn't me someone else would've asked later.

Reply:
Unity and some other 3D programs call them light cookies :slightly_smiling_face:

Reply:
Could add "gobo" as well. That's how Source engine calls it.